AlphaWallet/Settings/ViewControllers/AdvancedSettingsViewController.swift

```diff
@@ -35,12 +36,18 @@ class AdvancedSettingsViewController: UIViewController {
     }()
     private let roundedBackground = RoundedBackground()
     private var config: Config
+    private let keystore: Keystore
     weak var delegate: AdvancedSettingsViewControllerDelegate?
 
-    init(config: Config) {
+    init(keystore: Keystore, config: Config) {
         self.config = config
+        self.keystore = keystore
         super.init(nibName: nil, bundle: nil)
 
+        if keystore.currentWallet.isReal() {
```

Comment:
Can you help add a flag to `Features.swift` like `Features.isExportKeystoreJson`, set it to false, and check it here so the functionality is hidden?

I'll follow up with a PR so it works with HD wallets and enable it.